CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ __Notable Changes__
 * An easy way to include several edit mode related partials is now available (#1120):
   `render 'alchemy/edit_mode'` loads `menubar` and `preview_mode_code` at once
 * Add support for Turbolinks 5.0 (#1095)
+* Allow users to manually publish changes on global pages
```

Comment:
I removed it completely or do you prefer a second `unreleased` section in the README?